standard-questions.md

```diff
@@ -73,9 +74,9 @@ This document contains the set of questions that are being asked to assess each
             <td valign="top">Is there a mechanism for extracting or importing non PII data from the system in a non-proprietary format?</td>
             <td valign="top">
                 <ul>
-                    <li>Does this project have non-PII data?</li>
-                    <li>Is there a mechanism for extracting or importing non PII data from the system in a non-proprietary format?</li>
-                    <li>If yes - please describe the mechanism for extracting or importing data</li>
+                    <li>Does this project collect or use non-personally identifiable information (PII) data? </li>
+                    <li>If yes - is there a mechanism for extracting or importing non PII data from the system in a non-proprietary format?</li>
```

Comment:
Same here, please refer to `non-PII` (hyphenated for consistency with the above)

Reply:
Edited in 7e7156d